spring-kafka/src/main/java/org/springframework/kafka/listener/ContainerGroupSequencer.java

```diff
@@ -152,24 +152,22 @@ public synchronized void onApplicationEvent(ListenerContainerIdleEvent event) {
 	}
 
 	private synchronized void stopParentAndCheckGroup(MessageListenerContainer parent) {
-		if (parent.isRunning()) {
```

Comment:
Why do you remove this condition?

Reply:
I have removed the condition to verify `running` is true. That is not correct, since the status of the `ConcurrentContainer` would be set to `false` if one of the container is stopped. Earlier, it worked since `running` status of the `ConcurrentContainer` is not set to `false` if any of the container is stopped and it is not set to true after it is started.